python/ribasim/ribasim/config.py

```diff
@@ -185,20 +201,29 @@ class DiscreteControl(NodeModel):
         default_factory=TableModel[DiscreteControlLogicSchema]
     )
 
+    # TODO complete the ordering, and make sure it aligns with truth_state
+    _sort_keys: dict[str, list[str]] = {"static": ["node_id"], "logic": ["node_id"]}
```

Comment:
I tried to understand the code from a Julia point of view. DiscreteControlLogic seems to be coupled to a `StructVector`, which calls `sorted_table!`, which will `sort_by_function`. This function has multiple implementations. `DiscreteControlLogicV1` does not have a specific implementation. Therefore Julia will `sort_by_time_id`. So that's not the same as what is stated here. Correct?

Reply:
Ah, found something else. `sort_by_function(table::StructVector{<:Legolas.AbstractRecord}) = sort_by_id`
So, indeed they are sorted by node_id, just as mentioned in validation.jl